packages/language/lib/src/russian.dart

```diff
@@ -0,0 +1,18 @@
+import 'interface.dart';
+
+class RussianLocale implements Locale {
```

Comment:
`RussianLocale`  can be private?

Reply:
It actually is. But we can't hide object fields when dart is running.
This implementation is imported only to "library language".

Also, it can be public in the future, we must discuss it.
We plan that language will be switched from system settings.
But, we can also make possible a switch while app is running, and then we will need a constructor to give as new instance to lang.locale and it need to be public.